src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  const rent = days * cost;
```

Comment:
The variable name 'rent' is not clear enough. Rename it to 'totalRentCost' to indicate it's the total rent cost for the given days. 🌈